packages/census/utils.js

```diff
@@ -0,0 +1,32 @@
+const Os = Npm.require('os');
+
+Utils = {
+  // Sends a non-blocking request
+  request(method, url, options, cb) {
+    options = _.clone(options);
+
+    HTTP.call(method, url, options, Meteor.bindEnvironment((err, result) => {
+      // If attempts are still left, keep trying
+      if (err && options.attempts--)
+        Utils.request(options, cb);
+      // If request has been succeeded or no more attempts left, invoke callback
+      else
+        cb(err, result);
+    }));
+  },
+
+  // Gets the ip address
+  ip() {
```

Comment:
There is a chance someone who does not look at the forums or misses this issue will not know to do so.  I am against having this as an opt-out package, I do not feel that it is appropriate for MDG to add automatic statistic collection.  I feel this should be an opt-IN package, with only a notification post project creation to say this package exists, how to install it with `meteor add census`, an a link to the documentation/source.